ledger_device_sdk/src/nbgl/nbgl_home_and_settings.rs

```diff
@@ -190,4 +203,65 @@ impl<'a> NbglHomeAndSettings {
             }
         }
     }
+
+    /// Show the home screen and settings page.
+    /// This function returns immediately after the screen is displayed.
+    pub fn display(&mut self) {
```

Comment:
@yogh333 could be called `show_and_return` it would outline the difference with `show` a bit better imo.